backend/src/model/entity.rs

```diff
@@ -11,70 +11,230 @@ use diesel::{Identifiable, Insertable, Queryable};
-    pub water: Option<Water>,
-    pub sun: Option<Sun>,
+    ///
     pub shade: Option<String>,
+    /// The list of enum values.
     pub soil_ph: Option<Vec<Option<SoilPh>>>,
+    /// The list of enum values.
     pub soil_texture: Option<Vec<Option<SoilTexture>>>,
+    /// The list of enum values.
     pub soil_water_retention: Option<Vec<Option<SoilWaterRetention>>>,
+    ///
     pub environmental_tolerances: Option<Vec<Option<String>>>,
-    pub native_climate_zones: Option<String>,
-    pub adapted_climate_zones: Option<String>,
+    ///
     pub native_geographical_range: Option<String>,
+    ///
     pub native_environment: Option<String>,
+    ///
     pub ecosystem_niche: Option<String>,
-    pub root_zone_tendancy: Option<RootZoneTendancy>,
+    /// Enum value.
     pub deciduous_or_evergreen: Option<DeciduousOrEvergreen>,
+    /// Enum value.
     pub herbaceous_or_woody: Option<HerbaceousOrWoody>,
+    /// The list of enum values.
     pub life_cycle: Option<Vec<Option<LifeCycle>>>,
-    pub growth_rate: Option<GrowthRate>,
-    pub mature_size_height: Option<String>,
-    pub mature_size_width: Option<String>,
+    /// The list of enum values.
+    pub growth_rate: Option<Vec<Option<GrowthRate>>>,
+    ///
+    pub height: Option<String>,
+    ///
+    pub width: Option<String>,
+    /// The list of enum values.
     pub fertility: Option<Vec<Option<Fertility>>>,
+    ///
     pub pollinators: Option<String>,
+    ///
     pub flower_colour: Option<String>,
+    /// Enum value.
     pub flower_type: Option<FlowerType>,
+    /// The creation date of the entry.
     pub created_at: NaiveDateTime,
+    /// The last update date of the entry.
     pub updated_at: NaiveDateTime,
+    ///
     pub has_drought_tolerance: Option<bool>,
+    ///
     pub tolerates_wind: Option<bool>,
+    /// The list of the references of the plant.
     pub plant_references: Option<Vec<Option<String>>>,
+    /// Boolean value indicating whether the plant is a tree. The initial value is to `True` if  herbaceous_or_woody (woody) and life_cycle (perennial)
     pub is_tree: Option<bool>,
+    /// Enum value. The initial value is to `light feeder` if "Nutritionally poor soil" in `environmental_tolerances` is present.
     pub nutrition_demand: Option<NutritionDemand>,
+    /// Number value between -1..6 (-1 should be printed as 00)
     pub preferable_permaculture_zone: Option<i16>,
+    /// Date value fetched from PracticalPlants page showing the last modification date of the plant.
     pub article_last_modified_at: Option<NaiveDateTime>,
+    ///
+    pub hardiness_zone: Option<String>,
+    /// The list of enum values.
+    pub light_requirement: Option<Vec<Option<LightRequirement>>>,
+    /// The list of enum values.
+    pub water_requirement: Option<Vec<Option<WaterRequirement>>>,
+    /// The list of enum values.
+    pub propagation_method: Option<Vec<Option<PropagationMethod>>>,
+    ///
+    pub alternate_name: Option<String>,
```

Comment:
Is this an alternate Latin name? Please add more comments.

Reply:
I'm actually not sure what is the difference between the `name`, which is used during the merge with the `common_name_en`, and the `alternate_name` column. Both value sets are coming from permapeople but have different values. Maybe Yvonne can provide us with some insights into these values once she sends me the biological part of the descriptions for the columns